deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/Layer.java

```diff
@@ -97,6 +101,20 @@ protected void initializeConstraints(Builder<?> builder) {
         this.iDropout = builder.iDropout;
     }
 
+
+    /**
+     * Define the layer for SameDiff conversion
+     *
+     * @param sameDiff SameDiff instance
+     * @param layerInput Input to the layer
+     * @param paramTable Parameter table - keys and shapes as defined in the layer implementation class.
+     * @param mask Optional, maybe null. Mask to apply if supported
+     * @return The final layer variable corresponding to the activations/output from the forward pass
+     */
+    public SDVariable defineLayer(@NonNull SameDiff sameDiff, @NonNull SDVariable layerInput, @NonNull Map<String, SDVariable> paramTable, SDVariable mask){
+        throw new UnsupportedOperationException("SameDiff conversion has not been implemented for " + this.getClass().getSimpleName());
```

Comment:
Let's reorder this signature:
```
defineLayer(@NonNull SameDiff sameDiff, @NonNull SDVariable layerInput, SDVariable mask, @NonNull Map<String, SDVariable> paramTable)
```

Reply:
I copied it from `SameDiffLayer`, should I do the same there?